ipaserver/plugins/cert.py

```diff
@@ -1469,18 +1451,17 @@ def _ldap_search(self, all, raw, pkey_only, no_members, **options):
                     try:
                         obj = result[key]
                     except KeyError:
-                        obj = self._get_cert_obj(cert, all, raw, pkey_only)
+                        obj = {}
```

Comment:
Indeed, this is a bug. I added a user certificate. Now, when I do `ipa cert-find --pkey-only`, the result is:
```
-----------------------
11 certificates matched
-----------------------

...

-----------------------------
Number of entries returned 11
-----------------------------
```
but only 10 certificates are shown.

I would also expect that when I specify `--pkey-only` only primary keys are shown, which is not true but I suspect it's behaved like this for quite some time - only few certificates are displayed with limited information with the previous implementation. The `--pkey-only` option does not seem to have any effect with this patch, though, except that some certificates are not displayed at all.